src/common/config/env.validation.ts

```diff
@@ -277,6 +277,14 @@ export class EnvironmentVariables {
 
   @IsEnum(WorkingMode)
   public WORKING_MODE = WorkingMode.Finalized;
+
+  /**
+   * Effective balance.
+   */
+  @IsNumber()
+  @Min(32)
+  @Transform(({ value }) => parseInt(value, 10), { toClassOnly: true })
+  public EFFECTIVE_BALANCE_INCREMENTS = 32;
```

Comment:
Sorry for misleading you. Let's think about naming this var more concretely. For example, `REFERENCE_EFFECTIVE_BALANCE_INCREMENTS`

Reply:
Please, don't forget to add this var to `README.md`

Reply:
What do you think to use `MAX_EFFECTIVE_BALANCE`?
- ref. https://ethresear.ch/t/increase-the-max-effective-balance-a-modest-proposal/15801